### PR TITLE
Allow the first result for a search to be a non user and include the type in the returned data

### PR DESCRIPTION
**Pull request by mitchcapper**
Discussion in #459

### CODE REVIEW

**Comment by Schmavery** [comment]
Can you document the `type` field, with maybe a couple of examples of common values?

**Comment by mitchcapper** [comment]
Done. There was no documentation for the array at all, so expanded it.  Note getuserinfo returns the same type variable let me know if you want the options noted there as well.  Note I didn't find an official list of types, so just ran a bunch of queries to find what I could.

File: README.md [hunk 1]
```diff
@@ -227,3 +227,4 @@ login({appState: JSON.parse(fs.readFileSync('appstate.json', 'utf8'))}, (err, ap
 - [Klink](https://github.com/KeNt178/klink) - This Chrome extension will 1-click share the link of your active tab over Facebook Messenger
 - [Botyo](https://github.com/ivkos/botyo) - Modular bot designed for group chat rooms on Facebook
 - [matrix-puppet-facebook](https://github.com/matrix-hacks/matrix-puppet-facebook) - A facebook bridge for [matrix](https://matrix.org)
+- [facebot](https://github.com/Weetbix/facebot) - A facebook bridge for Slack.
```
Comment (Schmavery) [inline README.md]: Some merge problem?

**Comment by Schmavery** [comment]
Great, yeah anywhere we change the object should be noted (i.e. getUserInfo if applicable)

**Comment by mitchcapper** [comment]
I didn't change getUserInfo it already had the "type" property it just doesn't document what type is or any of the other properties it has.

I did add a reference to facebot in this commit, do you want a separate PR for that or just not at all?

**Comment by Schmavery** [comment]
Ah ok.  Can you add it then? haha 😛 

Ok.  We can probably just leave the facebook bit in, next time separate PR would be even better, but I won't tell anyone haha

**Comment by mitchcapper** [comment]
done.

**Pull request merged**